espnet/nets/asr_interface.py

```diff
@@ -1,15 +1,38 @@
 """ASR Interface module."""
+import argparse
+
+from espnet.bin.asr_train import get_parser
 from espnet.utils.dynamic_import import dynamic_import
+from espnet.utils.fill_missing_args import fill_missing_args
 
 
-class ASRInterface(object):
+class ASRInterface:
     """ASR Interface for ESPnet model implementation."""
 
     @staticmethod
     def add_arguments(parser):
         """Add arguments to parser."""
         return parser
 
+    @classmethod
+    def build(cls, idim: int, odim: int, **kwargs):
+        """Initialize this class with python-level args.
+
+        Args:
+            idim (int): The number of an input feature dim.
+            odim (int): The number of output vocab.
+
+        Returns:
+            ASRinterface: A new instance of ASRInterface.
+
+        """
+        args = argparse.Namespace(**kwargs)
+        args = fill_missing_args(
+            args, get_parser,
+            dummy_input="--outdir /tmp --dict /tmp/dummy.txt".split())
```

Comment:
It seems that this line will overwrite **kwags.

Reply:
no. I added tests
https://github.com/ShigekiKarita/espnet/blob/train-dtype/test/test_asr_interface.py#L10-L13